packages/mui-base/src/Popover/Positioner/usePopoverPositioner.tsx

```diff
@@ -26,7 +28,7 @@ export function usePopoverPositioner(
     renderedSide,
     renderedAlignment,
     positionerContext,
-  } = useAnchorPositioning(params);
+  } = useAnchorPositioning({ ...params, mounted });
```

Comment:
Destructuring the object manually could be faster at runtime.

Reply:
What do you mean destructuring? To avoid the spread here, it'd need to be part of `params` (which is doable but not really necessary)

Reply:
It looks like `mounted` already exists in the params for some other components, so changing to that for consistency

Reply:
I meant avoiding the `...` operator and creating the object (as per https://www.notion.so/mui-org/Performance-guide-memory-allocations-134cbfe7b66080cba325db614278fc20?pvs=4#134cbfe7b66080cc998de00ce75070a7), so
```
useAnchorPositioning({
  anchor: params.anchor,
  side: params.side,
  // ...
  mounted
});
```

Reply:
I refactored it so that `mounted` now exists in params so it can avoid the spread operator (some other components already had it like that)